src/operator/contrib/multibox_detection.cu

```diff
@@ -51,7 +51,9 @@ __device__ void CalculateOverlap(const DType *a, const DType *b, DType *iou) {
 }
 
 template<typename DType>
-__global__ void DetectionForwardKernel(DType *out, const DType *cls_prob,
+__global__
+__launch_bounds__(cuda::kMaxThreadsPerBlock)
```

Comment:
Can you explain why Volta need this?

Reply:
Kernel will fail to launch, too many resources.